micromasters/settings.py

```diff
@@ -374,9 +383,21 @@ def get_var(name, default):
         'elasticsearch': {
             'level': ES_LOG_LEVEL,
         },
+        'raven': {
+            'level': 'DEBUG',
+            'handlers': []
+        }
     },
 }
 
+# Sentry
+ENVIRONMENT = os.environ.get('MICROMASTERS_ENVIRONMENT', 'dev')
```

Comment:
The environment flag is a way of tracking the stage of the pipeline that an error occurs in for Sentry. I'm not sure what you mean about using `DEBUG`.